gsitemap.php

```diff
@@ -605,6 +620,7 @@ protected function _getSupplierLink(&$link_sitemap, $lang, &$index, &$i, $id_sup
             $supplier = new Supplier((int)$supplier_id['id_supplier'], $lang['id_lang']);
             $url = $link->getSupplierLink($supplier, $supplier->link_rewrite, $lang['id_lang']);
 
+            $image_link = $this->context->link->getBaseLink().Tools::getMediaServer(_THEME_SUP_DIR_)._THEME_SUP_DIR_.((!file_exists(_THEME_SUP_DIR_.'/'.(int)$supplier->id.'-'.ImageType::getFormattedName('medium'))) ? $lang['iso_code'].'-default' : (int)$supplier->id).'-'.ImageType::getFormattedName('medium');
```

Comment:
Same comment about `$this->context->link->getBaseLink()`

Reply:
Please check this commit. I removed supplier and manufacturer because they are not used as pages I think?

Reply:
https://github.com/PrestaShop/gsitemap/pull/53/commits/91de6e87abad28c80692137bab532cf1e392ee13